tests/should_succeed/mathlibtest.jou

```diff
@@ -0,0 +1,37 @@
+# This is a math lib test for math lib
+# It includes abs, llabs, fabs, ceil, floor, round, acos, asin, atan, atan2, cos, sin, tan,
+#             fmod, sqrt, cbrt, log, log10, log2, exp, exp2, pow, ldexp, fmax, fmin, hypot
+from "stdlib/math.jou" import (
+    abs, llabs, fabs, ceil, floor, round, acos, asin, atan, atan2, cos, sin, tan,
+    fmod, sqrt, cbrt, log, log10, log2, exp, exp2, pow, ldexp, fmax, fmin, hypot
+    )
+from "stdlib/io.jou" import printf
+
+def main() -> int:
+    printf("%d\n", abs(-1)) # Output: 1
+    printf("%lld\n", llabs(-1145141919180L)) # Output: 1145141919180
+    printf("%f\n", fabs(-114.514)) # Output: 114.514000
+    printf("%f\n", ceil(3.14159265358979)) # Output: 4.000000
+    printf("%f\n", floor(3.14159265358979)) # Output: 3.000000
+    printf("%f\n", round(3.14159265358979)) # Output: 3.000000
+    printf("%f\n", acos(-1)) # Output: 3.141593
+    printf("%f\n", asin(-1)) # Output: -1.570796
+    printf("%f\n", atan(-1)) # Output: -0.785398
+    printf("%f\n", atan2(-1, -1)) # Output: -2.356194
+    printf("%f\n", cos(45)) # Output: 0.525322
+    printf("%f\n", sin(45)) # Output: 0.850904
+    printf("%f\n", tan(90)) # Output: -1.995200

```

Comment:
Please change the values to something else than 45, 45, 90. These functions take the angles as radians, so `cos(45)` is **not** the cosine of 45 degrees (which is sqrt(2)/2 so approximately 0.7, not 0.52).

Reply:
changed